tls/s2n_resume.c

```diff
@@ -242,7 +239,25 @@ static S2N_RESULT s2n_tls13_client_deserialize_session_state(struct s2n_connecti
     return S2N_RESULT_OK;
 }
 
-static S2N_RESULT s2n_client_deserialize_session_state(struct s2n_connection *conn, struct s2n_stuffer *from)
+static S2N_RESULT s2n_deserialize_resumption_state(struct s2n_connection *conn, struct s2n_blob *identity, struct s2n_stuffer *from)
+{
+    RESULT_ENSURE_REF(conn);
+    RESULT_ENSURE_REF(from);
```

Comment:
We could also add this null check here:  `RESULT_ENSURE_REF(identity);`

Reply:
Actually identity can be null, because in the TLS1.2 case we do not need to set a psk identity.